src/main/java/cubicchunks/worldgen/dependency/DependencyManager.java

```diff
@@ -36,139 +36,92 @@
 import java.util.Map;
 import java.util.Set;
 
+/*
+ * TODO: Abstraction to allow for non-cube dependents.
+ * TODO: Commenting
```

Comment:
Non-cube dependents are not needed right now, so it can wait. Is issue #14 going to be fixed? This is a critical issue for the dependency system.